manifests/p/PatrickSiegler/PrismatikUnofficial/5.11.2.31/PatrickSiegler.PrismatikUnofficial.locale.en-US.yaml

```
@@ -5,8 +5,24 @@ PackageIdentifier: PatrickSiegler.PrismatikUnofficial
 PackageVersion: 5.11.2.31
 PackageLocale: en-US
 Publisher: Patrick Siegler
+PublisherUrl: https://github.com/psieg
+Author: Patrick Siegler
 PackageName: Prismatik (unofficial)
+PackageUrl: https://github.com/psieg/Lightpack/releases/tag/5.11.2.31
```

```suggestion
PackageUrl: https://github.com/psieg/Lightpack
```
It would be better to keep it like this so that this package can be automated in the future.